docs/config.rst

```diff
@@ -289,6 +289,75 @@ Base options
    tox invocation environment it is ignored. The list of environment variable names is not case sensitive, for example:
    passing ``A`` or ``a`` will pass through both ``A`` and ``a``.
 
+   Some variables are always passed through to ensure the basic functionality of standard library functions or tooling like
+   pip. This is also not case sensitive on all platforms except Windows.
+
+    .. list-table:: Environment Variables per Operating System
+        :widths: 25 25 25 25
+        :header-rows: 1
+
+        *   - Environment Variable
+            - Linux
+            - MacOS
+            - Windows
```

Comment:
Missing PIP_* and VIRTUALENV_* , plus https://github.com/tox-dev/tox/blob/main/src/tox/tox_env/python/api.py#L133